mkt/extensions/tests/test_validation.py

```diff
@@ -50,10 +50,10 @@ def assertValidationError(self, key, **kwargs):
             if kwargs:
                 expected_message = expected_message % kwargs
             eq_(e.__class__, ParseError)
-            eq_(e.detail['key'], key)
-            eq_(e.detail['message'], expected_message)
+            eq_(e.detail['detail']['key'], key)
+            eq_(e.detail['detail']['message'], expected_message)
             if kwargs:
-                eq_(e.detail['params'], kwargs)
+                eq_(e.detail['detail']['params'], kwargs)
```

Comment:
This is a backwards-incompatible change, we should check the content tools to see if they cope correctly with that